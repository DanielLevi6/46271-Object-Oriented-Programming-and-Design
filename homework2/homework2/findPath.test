# findPath test file

# Create a graph
CreateGraph myGraph

# Add nodes
CreateNode node1 4
CreateNode node2 3
CreateNode node3 6
CreateNode node4 1

AddNode myGraph node1
AddNode myGraph node2
AddNode myGraph node3
AddNode myGraph node4

# Add an edge
AddEdge myGraph node1 node2
AddEdge myGraph node1 node3
AddEdge myGraph node2 node3
AddEdge myGraph node2 node4
AddEdge myGraph node3 node4
AddEdge myGraph node4 node4

# Find a path
FindPath myGraph node1 -> node4
