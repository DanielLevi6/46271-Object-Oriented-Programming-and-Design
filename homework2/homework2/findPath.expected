# findPath test file

# Create a graph
created graph myGraph

# Add nodes
created node node1 with cost 4
created node node2 with cost 3
created node node3 with cost 6
created node node4 with cost 1

added node node1 to myGraph
added node node2 to myGraph
added node node3 to myGraph
added node node4 to myGraph

# Add an edge
added edge from node1 to node2 in myGraph
added edge from node1 to node3 in myGraph
added edge from node2 to node3 in myGraph
added edge from node2 to node4 in myGraph
added edge from node3 to node4 in myGraph
added edge from node4 to node4 in myGraph

# Find a path
shortest path in myGraph: node1 node2 node4
